docs/reference/cat/nodes.asciidoc

```diff
@@ -314,6 +314,12 @@ Time spent in suggest, such as `0`.
 
 `suggest.total`, `suto`, `suggestTotal`::
 Number of suggest operations, such as `0`.
+
+`mappings.total_count`, `mtc`, `mappingsTotalCount`::
+Number of mappings, including <<runtime,runtime>> and <<object,object>> fields.
+
+`mappings.total_estimated_overhead_in_bytes`, `mteoi`, `mappingsTotalEstimatedOverheadInBytes`::
```

Comment:
Unrelated to this PR but why `mteoi` and not `mteo`? I think we can fix this without BwC concerns if we do so before 8.5.0 is released.

Reply:
I can change it. Would adding a version 8.5.0 label to the PR be enough for it to be backported to 8.5.0?

Reply:
It needs the 8.5.0 label but backporting is a separate process. I'll help on the PR itself if needed.

Reply:
Fixed, and added a label for backporting, I hope this will help, right?

Reply:
Ah I was expecting a separate PR for that but I see you've done it here. I suspect it doesn't matter much since we want to backport these docs too, but the code change is a little time-critical since it must land before we cut the final 8.5.0 BC (we can fix the docs later). Also it's a different kind of change, really a `>non-issue` since it relates to an unreleased bug and the labelling starts to get a bit confusing if we combine things like this. Still this is good to go now. Labels look good.